src/scripts/components/ProfileCuration.js

```diff
@@ -0,0 +1,522 @@
+              <Text gray>
+                Wait until the committ period ends to reveal it.You will send it{' '}
+                <Strong purple>back</Strong> when the reveal period starts. If
+                you don’t send it back your vote is not going to be counted.
+              </Text>
+            </InfoStatusContent>
+            <ButtonColor>Send back your vote</ButtonColor>
+          </InfoStatus>
+
+          <InfoStatus>
+            <InfoStatusContent>
+              <InfoStatusTitle>
+                <SVGIcon
+                  color="green"
+                  icon="icon-check"
+                  height="18px"
+                  width="18px"
+                  margin="0 10px 0 0"
+                />
+                This video has been approved
+              </InfoStatusTitle>
+            </InfoStatusContent>
+            <Text gray>
+              The Paratii community supported the permanence of this video on
+              the platform. The video will continue to exist on the plataform
+              and is going to be available for new challenges in 6 days
+            </Text>
+          </InfoStatus>
+
+          <InfoStatus>
+            <InfoStatusContent>
+              <InfoStatusTitle>Voting</InfoStatusTitle>
+            </InfoStatusContent>
+            <Voting>
+              <VotingValuesWrapper>
+                <VotingValue>
+                  <Text textTransform="uppercase" small green>
+                    Support
+                  </Text>
+                  <Title big bold gray>
+                    65%
+                  </Title>
+                </VotingValue>
+                <VotingValue>
+                  <Text textTransform="uppercase" small pink>
+                    Oppose
+                  </Text>
+                  <Title big bold gray>
+                    35%
+                  </Title>
+                </VotingValue>
+              </VotingValuesWrapper>
+              <VotingBarWrapper>
+                <VotingBar current="65" total="100" />
+              </VotingBarWrapper>
+            </Voting>
+            <Text small gray>
+              Date of the challenge: 04/03/2018
+            </Text>
+          </InfoStatus>
+
+          <InfoStatus>
+            <InfoStatusContent>
+              <InfoStatusTitle>
+                <SVGIcon
+                  color="pink"
+                  icon="icon-close"
+                  height="18px"
+                  width="18px"
+                  margin="0 10px 0 0"
+                />
+                This video has been rejected
+              </InfoStatusTitle>
+            </InfoStatusContent>
+            <Text gray>
+              The Paratii community opposed the permanence of this video on the
+              platform. The video was deleted because most of our trustees
+              believe that this video broke one of our policies
+            </Text>
+          </InfoStatus>
+        </Sidebar>
+      </Wrapper>
+    )
+  }
```

Comment:
This component is **massive**. It would be very helpful, I think, if it were broken up into smaller components in different files, each representing a logical piece of the larger component. 

We have a few already huge component files like `Play.js` and `VideoOverlay.js` (mostly thanks to me ;P) and they are difficult to work with right now 😄